test/README.md

```diff
@@ -134,16 +134,16 @@ deactivate
 
 To make the basic test suite run quickly, we use "faked" binary files to test the **CVE mappings**.  However, we want to be able to test real files to test that the **signatures** work on real-world data.
 
-In #99, I've added a _file_test function (to match the existing _binary_test) that takes a url, and package name and a version, and downloads the file, runs the scanner against it, and makes sure it is the file that you've specified.  But we need more tests!
+In #99, I've added a _file_test function (to match the existing _binary_test) that takes a url, and product name and a version, and downloads the file, runs the scanner against it, and makes sure it is the file that you've specified.  But we need more tests!
```

Comment:
In my opinion, It should be `url, package name, module name and a version`, Because what we are downloading is linux package, we don't call it product.

Reply:
Then I guess it would be nice to change the name of the test_function to be _package_test rather than file test.